libs/skills/catalog/frontmcp-observability/references/metrics-endpoint.md

```
@@ -0,0 +1,161 @@
+```typescript
+import { createCounter } from '@frontmcp/observability';
+
+const cacheHits = createCounter('my_cache_hits_total', 'Cache hits by tier');
+
+// In a tool / provider:
+cacheHits.inc(1, { tier: 'l1' });
+```
+
+The next scrape will include:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)